addons/garage/XEH_preInit.sqf

```diff
@@ -6,6 +6,7 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+GVAR(currentTab) = -1;
```

Comment:
By default, why shouldn’t it start at 0?

Reply:
I think I chose this since BI garage initially opens with no tab selected.